feature-libs/quote/components/config/augmented-config.model.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { QuoteUIConfig } from './quote-ui.config';
+
+declare module '@spartacus/quote/root' {
```

Comment:
That needs to be 
```suggestion
declare module '@spartacus/core' {
```
because we augment an interface from core

Reply:
done